tests/query.rs

```diff
@@ -21,9 +21,19 @@ use predicates::prelude::*; // Used for writing assertions
 
 #[tokio::test]
 async fn test_simple_query() -> Result<(), Box<dyn std::error::Error>> {
-    let table_name = prepare_pk_sk_table().await?;
+    let mut tm = util::setup().await?;
```

Comment:
There is a common code. However, I think splitting code is more suitable in this situation. Fine as it is.